photon-server/build.gradle

```diff
@@ -146,15 +121,7 @@ run {
   environment "PATH_PREFIX", "../"
 }
 
-// task overrideToPi {
-//     doLast {
-//         project.setProperty('ArchOverride', 'linuxarm32')
-//     }
-// }
-
-
-
-task deploy {
+task deployIdc {
```

Comment:
Que

Reply:
This conflicts with GradleRIO's deploy task, we need to rename this to something better

Reply:
Why are we using gradlerio in photon code?

Reply:
Not sure, the standalone example did so to track dependency versions, might be a wrapper in native utils, not sure

Reply:
we do need the deploy task to work before I ship this, either option is good w me